src/wp-admin/css/common.css

```
@@ -1289,12 +1296,14 @@ th.action-links {
 }
 
 .filtered-by .tags {
-	display: inline;
+        display: flex;
+        align-items: flex-start;
+        flex-wrap: wrap;
+        gap: 3px;
```

```suggestion
	gap: 8px;
```

These tags had a gap of 10 pixels from left and right margins. That seems slightly high with the flexbox display, though I would like more than 3 pixels. If the padding remains `4px 8px`, I recommend an 8-pixel gap.